<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>969</width>
    <height>744</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Functional Training Group</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_23">
   <item row="0" column="0" rowspan="2" colspan="2">
    <layout class="QGridLayout" name="gridLayout_22">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_11">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0" colspan="2">
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>188</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="PushButtonGerarPlanilha">
             <property name="text">
              <string>Gerar Planilha</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QPushButton" name="PushButtonGerarRelatorio">
             <property name="text">
              <string>Gerar Relatório</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="PushButtonTotem">
             <property name="text">
              <string>Solicitar Informações do Totem</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="1" column="0">
              <widget class="QComboBox" name="ComboBoxNomes"/>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QSpinBox" name="spinBoxIDTotem">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>10</number>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QLabel" name="Atualizacoes">
               <property name="whatsThis">
                <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
               </property>
               <property name="styleSheet">
                <string notr="true">border-color: rgb(0, 168, 90);
border-width : 2px;
border-style: inset;

</string>
               </property>
               <property name="text">
                <string>Atualizações dos totens:</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QPushButton" name="PushButtonLimparAtualizacoes">
               <property name="text">
                <string>Limpar atualizações</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="1">
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="0">
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="0" column="0">
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QComboBox" name="comboBoxUSB"/>
               </item>
               <item row="1" column="0">
                <widget class="QPushButton" name="pushButtonUSB2">
                 <property name="text">
                  <string>Atualizar</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QPushButton" name="pushButtonUSB">
                 <property name="text">
                  <string>Conectar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="pushButtonBancoDados">
           <property name="text">
            <string>Enviar informações dos totens para o banco de dados</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QGridLayout" name="gridLayout_21">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_20">
         <item row="0" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="0">
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="0">
            <widget class="QCalendarWidget" name="calendarWidget">
             <property name="maximumSize">
              <size>
               <width>14777215</width>
               <height>14777215</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Cadastro de novos participantes</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_19">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_14">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_13">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Nome:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <widget class="QLineEdit" name="lineEditNome"/>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <layout class="QGridLayout" name="gridLayout_17">
              <item row="0" column="0">
               <layout class="QGridLayout" name="gridLayout_15">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEditIdade"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Idade:</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="0" column="1">
               <layout class="QGridLayout" name="gridLayout_16">
                <item row="0" column="0">
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item row="1" column="0">
                 <widget class="QPushButton" name="PushButtonCadastro">
                  <property name="text">
                   <string>Cadastrar</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="labelAvisoCadastro">
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>20</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>imagens/FTG.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>40</y>
        <width>391</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>imagens/LOGOS-20241002T010036Z-001/LOGOS/g3743.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>141</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>imagens/calistenia.jpeg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>731</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>                                    Desenvolvido pelo Grupo de Pesquisa em Instrumentação da UFS. 2024. </string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder>PushButtonGerarRelatorio</zorder>
  <zorder>PushButtonGerarPlanilha</zorder>
  <zorder>calendarWidget</zorder>
  <zorder>PushButtonTotem</zorder>
  <zorder>spinBoxIDTotem</zorder>
  <zorder>ComboBoxNomes</zorder>
  <zorder>Atualizacoes</zorder>
  <zorder>PushButtonLimparAtualizacoes</zorder>
  <zorder>lineEditNome</zorder>
  <zorder>label_7</zorder>
  <zorder>label_8</zorder>
  <zorder>lineEditIdade</zorder>
  <zorder>PushButtonCadastro</zorder>
  <zorder>labelAvisoCadastro</zorder>
  <zorder>comboBoxUSB</zorder>
  <zorder>pushButtonUSB</zorder>
  <zorder>pushButtonUSB2</zorder>
  <zorder>pushButtonBancoDados</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>horizontalSpacer_3</zorder>
  <zorder>verticalSpacer_2</zorder>
  <zorder>verticalSpacer_3</zorder>
  <zorder>groupBox</zorder>
  <zorder>horizontalSpacer_2</zorder>
  <zorder>horizontalSpacer_5</zorder>
  <zorder>horizontalSpacer_6</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>frame</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
